symengine/printer.cpp

```diff
@@ -124,6 +124,47 @@ void StrPrinter::bvisit(const Interval &x)
     str_ = s.str();
 }
 
+void StrPrinter::bvisit(const BooleanAtom &x)
+{
+    if (x.get_val()) {
+        str_ = "True";
+    } else {
+        str_ = "False";
+    }
+}
+
+void StrPrinter::bvisit(const Contains &x)
+{
+    std::ostringstream s;
+    s << "Contains(" << apply(x.get_expr()) << ", " << apply(x.get_set())
+      << ")";
+    // s << apply(x.get_expr()) << " " << "∈" << " " << apply(x.get_set());
```

Comment:
That's an alternative way of printing. That's what should be done for a pretty printer, not the string printer. I'll remove it as it is not used.